numpy/lib/function_base.py

```diff
@@ -2921,10 +2921,13 @@ def i0(x):
 
     See Also
     --------
-    scipy.special.iv, scipy.special.ive
+    scipy.special.i0, scipy.special.iv, scipy.special.ive
 
     Notes
     -----
+    The scipy implementation is recommended over this function:
```

Comment:
Can you do an intersphinx link here somehow?

Reply:
That would be under `See also`, I guess. All but two mentions of scipy now lack this, so let's leave this for a more general docstring cleanup.